q2_diversity/plugin_setup.py

```diff
@@ -18,18 +22,68 @@
     package='q2_diversity'
 )
 
-# TODO create decorator for promoting functions to methods. This info would
-# be moved to the decorator calls.
 plugin.methods.register_function(
-    function=q2_diversity.beta_diversity,
+    function=q2_diversity.beta_phylogenetic,
     # TODO require a uniform sampling effort FeatureTable when predicates exist
     inputs={'feature_table': FeatureTable[Frequency],
-            # TODO this is optional; how do we handle that here?
             'phylogeny': Phylogeny},
-    parameters={'metric': Str},
+    parameters={'metric': Str % Choices(beta.phylogenetic_metrics())},
+    outputs=[('distance_matrix', DistanceMatrix % Properties('phylogenetic'))],
+    name='Beta diversity',
```

Comment:
update name to indicate it's phylogenetic bdiv, to distinguish from `beta` name below